lib/new_relixir/statman.ex

```diff
@@ -0,0 +1,186 @@
+defmodule NewRelixir.Statman do
+  def poll do
+    [end_time, start_time, metrics, errors] = NewRelixir.Collector.poll
+    transform_aggregated_metrics(metrics, errors, {start_time, end_time})
+  end
+
+  def transform_aggregated_metrics(metrics, errors, time) do
+    ms = metrics
```

Comment:
Since the map itself is enumerable and the output format of `transform_metric` is now standardized, we can simplify this and also skip the filter step:

```elixir
# Before
ms = metrics
      |> Map.to_list
      |> Enum.reduce([], fn(m, acc) ->
        acc ++ transform_metric(m)
      end)
      |> Enum.filter(&(&1 != []))

# After
ms = Enum.flat_map(metrics, &transform_metric/1)
```

Reply:
Done in def81565faac1b97b15e0d9e48a4ad8892c40848